src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -94,11 +92,7 @@ public PSRoleDefinition GetRoleDefinition(Guid roleId, string scope)
         public IEnumerable<PSRoleDefinition> FilterRoleDefinitions(string name, string scope, ulong first = ulong.MaxValue, ulong skip = 0)
         {
             Rest.Azure.OData.ODataQuery<RoleDefinitionFilter> odataFilter = new Rest.Azure.OData.ODataQuery<RoleDefinitionFilter>(item => item.RoleName == name);
-            return new GenericPageEnumerable<RoleDefinition>(
-                delegate ()
-                {
-                    return AuthorizationManagementClient.RoleDefinitions.List(scope, odataFilter);
-                }, AuthorizationManagementClient.RoleDefinitions.ListNext, first, skip)
+            return AuthorizationManagementClient.RoleDefinitions.List(scope, odataFilter)
```

Comment:
@darshanhs90 a couple of questions surrounding this change:

- Do we need to make this change to all of the instances of paging in this file?
- Which cmdlets are we seeing performance issues with?
- This change could break users who are currently getting all pages of definitions or assignments and querying over those. How should we handle that?